python/semantic_kernel/functions/old/context_variables.py

```diff
@@ -0,0 +1,126 @@
+# # Copyright (c) Microsoft. All rights reserved.
```

Comment:
Are we leaving the "old" dir code in until your branch is merged to the feature and we clean up the samples?